.gitattributes

```diff
@@ -13,8 +13,8 @@
 *.isma binary
 
 *.cs text diff=csharp
-*.csproj text merge=union
-*.sln text eol=crlf merge=union
+*.csproj text=auto
+*.sln text=auto eol=crlf
```

Comment:
I'm wondering if .slns should be kept as unions, though. Should almost never be line change conflicts, but just added or removed lines.

Reply:
Possibly but given I don't know what sort of issues this might cause for example if someone reorders projects I'm not sure it is worth it. I prefer to not have the extra magic and have folks manually merge instead of accept both which is what this essentially does. Plus I figure we would just follow the lead of the .NET team as I'm sure they have dealt with more issues in this space.